gpii/node_modules/settingsHandlers/test/JawSettingsHandlerTests.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
```

Comment:
Can we rename this file to JawsSettingsHandlerTests.js?

Reply:
Renamed.